x-pack/docs/en/security/auditing/output-logfile.asciidoc

```diff
@@ -3,13 +3,39 @@
 === Logfile audit output
 
 The `logfile` audit output is the default output for auditing. It writes data to
-the `<clustername>_access.log` file in the logs directory.
+the `<clustername>_audit.log` file in the logs directory. To maintain
+compatibility with releases prior to 6.5.0, another file `<clustername>_access.log`
+is also generated next to it. They differ in the output format but the contents
```

Comment:
Not a big deal, but I suggest changing this to:
"...release prior to 6.5.0, a `<clustername>_access.log` file is also generated."